zulipterminal/ui_tools/buttons.py

```diff
@@ -118,6 +118,18 @@ def __init__(self, controller: Any, width: int) -> None:
                          count=0)  # Starred messages are already marked read
 
 
+class MentionedButton(TopButton):
+    def __init__(self, controller: Any, width: int, count: int=0) -> None:
+        button_text = ("Mentions [" +
+                       keys_for_command("ALL_MENTIONS").pop() +
+                       "]")
+        super().__init__(controller, button_text,
+                         controller.show_all_mentions,
+                         width=width,
+                         count=count,
+                         prefix_character='')
```

Comment:
There's no test for this, but this seems reasonable for now given the simplicity :)